notes-to-self/graceful-shutdown-idea.py

```diff
@@ -31,9 +33,11 @@ async def stream_handler(stream):
     while True:
         with gsm.cancel_on_graceful_shutdown():
             data = await stream.receive_some()
+            print(f"{data = }")
```

Comment:
what's up with this random added print?

Reply:
Otherwise the flake8 says variable is unused, and I thought it would be better to use the data than remove the variable assignment. Up to you what you think best action is.

Reply:
sounds good :+1: